.github/workflows/pr.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       - name: Fetch Sources
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c #v3.3.0
 
-      # Setup Java 11 environment for the next steps
+      # Setup Java 17 environment for the next steps
```

Comment:
![Screenshot 2023-06-19 at 16 31 58](https://github.com/apollographql/apollo-kotlin/assets/3974977/a46f86e8-45f3-4088-8efa-1fbc23cb94ef)

Might as well remove this comment at this point?

Reply:
hahahaha 😂. Yes 💯